generator/src/routes/+page.svelte

```diff
@@ -61,33 +172,66 @@
         </div>
       </div>
       <div class="actions-wrapper">
-        <Button title="Generate IOXIO Tag" onClick={() => {}} />
+        <Button disabled={status === Status.GENERATING} title="Generate IOXIO Tag" type="submit" />
       </div>
     </form>
   </div>
   <!-- Right Panel -->
   <div class="qrcode-area-wrapper">
     <div class="qrcode-area">
-      <div class="qrcode">
-        <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+      <div class="qrcode-frame">
+        {#if status !== Status.GENERATING}
+          <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+          <img class="logo" src={IoxioTagsLogo} alt="" aria-hidden="true" />
+          {#if status === Status.FINISHED}
+            <img class="qrcode" alt="" aria-hidden="true" src="" bind:this={qrcodeElement} />
+            <img class="ioxio-tag-frame" src={IoxioTagsQrcodeFrame} alt="" aria-hidden="true" />
```

Comment:
Please stop adding this, I've *JUST* told you that it will be added by the backend.